pages/_app.tsx

```diff
@@ -63,7 +62,7 @@ function MyApp({ Component, pageProps }: AppProps) {
               clientId={process.env.NEXT_PUBLIC_TINA_CLIENT_ID}
               {...pageProps}
             >
-              {(livePageProps: TinaCMSProviderProps) => <Component {...livePageProps} />}
+              {(livePageProps: any) => <Component {...livePageProps} />}
```

Comment:
Unfortunately I don't think that `TinaCMSProviderProps` is the correct type here. It can really be anything returned in `getStaticProps`.